src/adapter/src/catalog/migrate.rs

```diff
@@ -72,12 +74,18 @@ pub(crate) async fn migrate(
 
     info!("migrating from catalog version {:?}", catalog_version);
 
+    // In v0.82.0 we changed the 'default' cluster to be named 'quickstart'. For all _existing_
```

Comment:
Why did this move?

Reply:
`Catalog::load_catalog_items` plans the items from the txn, so I wanted to ensure that they were planned with the right default cluster (this migration goes from `quickstart` to `default`), and my PR introduces a new usage of the session's cluster––if you don't specify `IN CLUSTER`, that's where the object gets scheduled. It was more defensive than anything.

The release is going out without this change, though, so it's a moot point. Will remove this from my branch.